src/pom.xml

```diff
@@ -1287,7 +1287,7 @@
     <dependency>
      <groupId>com.github.oshi</groupId>
      <artifactId>oshi-core</artifactId>
-     <version>5.3.3</version>
+     <version>5.4.0</version>
```

Comment:
So much reformatting changes... I guess this one is the only actual change?

Been itching to clean up the maven files for a while... we've been over it already a couple of months ago:
https://github.com/geotools/geotools/pull/3201#discussion_r519682231

Reply:
ahhh.. I have "remove training spaces" checked somewhere. I can revert the whitespace checks if you like?

Reply:
Euh, I prefer clean PRs, but as long as this is backported on all active branches I don't mind so much as to ask to be cleanedup it. More motivation for me to look into XML formatting plugins.